--- chef-11.8.0/lib/chef/handler/json_file.rb~	2013-12-21 17:25:17.000000000 +0200
+++ chef-11.8.0/lib/chef/handler/json_file.rb	2013-12-21 17:25:58.665209433 +0200
@@ -27,7 +27,7 @@
 
       def initialize(config={})
         @config = config
-        @config[:path] ||= "/var/chef/reports"
+        @config[:path] ||= "/var/lib/chef/reports"
         @config
       end
 
--- chef-12.10.24/chef-config/lib/chef-config/config.rb~	2016-06-02 00:23:49.000000000 +0300
+++ chef-12.10.24/chef-config/lib/chef-config/config.rb	2016-06-02 00:24:58.502867530 +0300
@@ -170,7 +170,7 @@
 
     # Location of cookbooks on disk. String or array of strings.
     # Defaults to <chef_repo_path>/cookbooks.  If chef_repo_path
-    # is not specified, this is set to [/var/chef/cookbooks, /var/chef/site-cookbooks]).
+    # is not specified, this is set to [/var/lib/chef/cookbooks, /var/lib/chef/site-cookbooks]).
     default(:cookbook_path) do
       if self.configuration[:chef_repo_path]
         derive_path_from_chef_repo_path("cookbooks")
@@ -234,10 +234,10 @@
       if local_mode
         PathHelper.join(config_dir, "local-mode-cache")
       else
-        primary_cache_root = platform_specific_path("/var")
-        primary_cache_path = platform_specific_path("/var/chef")
-        # Use /var/chef as the cache path only if that folder exists and we can read and write
-        # into it, or /var exists and we can read and write into it (we'll create /var/chef later).
+        primary_cache_root = platform_specific_path("/var/cache")
+        primary_cache_path = platform_specific_path("/var/cache/chef")
+        # Use /var/cache/chef as the cache path only if that folder exists and we can read and write
+        # into it, or /var exists and we can read and write into it (we'll create /var/cache/chef later).
         # Otherwise, we'll create .chef under the user's home directory and use that as
         # the cache path.
         unless path_accessible?(primary_cache_path) || path_accessible?(primary_cache_root)
