--- chef-11.8.0/lib/chef/config.rb~	2013-12-21 17:23:33.000000000 +0200
+++ chef-11.8.0/lib/chef/config.rb	2013-12-21 17:24:18.980654855 +0200
@@ -162,7 +162,7 @@
           end
         end
       else
-        platform_specific_path("/var/chef")
+        platform_specific_path("/var/lib/chef")
       end
     end
 
@@ -260,7 +260,7 @@
       if local_mode
         "#{config_dir}local-mode-cache"
       else
-        platform_specific_path("/var/chef")
+        platform_specific_path("/var/cache/chef")
       end
     end
 
--- chef-11.8.0/lib/chef/config.rb~	2013-12-21 17:25:17.000000000 +0200
+++ chef-11.8.0/lib/chef/config.rb	2013-12-21 17:25:40.044358776 +0200
@@ -201,7 +201,7 @@
 
     # Location of cookbooks on disk. String or array of strings.
     # Defaults to <chef_repo_path>/cookbooks.  If chef_repo_path
-    # is not specified, this is set to [/var/chef/cookbooks, /var/chef/site-cookbooks]).
+    # is not specified, this is set to [/var/lib/chef/cookbooks, /var/lib/chef/site-cookbooks]).
     default(:cookbook_path) do
       if self.configuration[:chef_repo_path]
         derive_path_from_chef_repo_path('cookbooks')
--- chef-11.8.0/lib/chef/handler/json_file.rb~	2013-12-21 17:25:17.000000000 +0200
+++ chef-11.8.0/lib/chef/handler/json_file.rb	2013-12-21 17:25:58.665209433 +0200
@@ -27,7 +27,7 @@
 
       def initialize(config={})
         @config = config
-        @config[:path] ||= "/var/chef/reports"
+        @config[:path] ||= "/var/lib/chef/reports"
         @config
       end
 
