--- chef-12.10.24/chef.gemspec	2016-06-02 01:34:04.243782081 +0300
+++ chef-12.10.24/chef.gemspec	2016-06-02 19:54:04.001988520 +0300
@@ -31,10 +31,6 @@
   s.add_dependency "erubis", "~> 2.7"
   s.add_dependency "diff-lcs", "~> 1.2", ">= 1.2.4"
 
-  s.add_dependency "chef-zero", "~> 4.5"
-
-  s.add_dependency "plist", "~> 3.2"
-
   # Audit mode requires these, so they are non-developmental dependencies now
   %w{rspec-core rspec-expectations rspec-mocks}.each { |gem| s.add_dependency gem, "~> 3.4" }
   s.add_dependency "rspec_junit_formatter", "~> 0.2.0"
