--- chef-14.1.21/spec/unit/provider_resolver_spec.rb.orig	2018-05-21 18:22:54.000000000 +0200
+++ chef-14.1.21/spec/unit/provider_resolver_spec.rb	2018-07-14 21:52:07.158840235 +0200
@@ -722,6 +722,16 @@
             },
           },
 
+          "pld" => {
+    #        service: [ Chef::Resource::SystemdService, Chef::Provider::Service::Systemd ],
+            package: [ Chef::Resource::PoldekPackage, Chef::Provider::Package::Poldek ],
+            ifconfig: [ Chef::Resource::Ifconfig, Chef::Provider::Ifconfig::Redhat ],
+            group:   [ Chef::Resource::Group, Chef::Provider::Group::Usermod ],
+
+            "pld" => {
+              "3.0" => {
+              },
+            },
         },
 
         "freebsd" => {
