--- chef-11.14.2/lib/chef/platform/provider_mapping.rb~	2014-12-04 18:02:46.000000000 +0200
+++ chef-11.14.2/lib/chef/platform/provider_mapping.rb	2014-12-04 18:07:26.765336890 +0200
@@ -194,6 +194,16 @@
                 :group => Chef::Provider::Group::Suse
               }
             },
+            :pld => {
+              :default => {
+                :service => Chef::Provider::Service::Redhat,
+                :cron => Chef::Provider::Cron,
+                :package => Chef::Provider::Package::Poldek,
+                :mdadm => Chef::Provider::Mdadm,
+                :ifconfig => Chef::Provider::Ifconfig::Redhat, # TODO: PLD config path needed
+                :group => Chef::Provider::Group::Usermod,
+              }
+            },
             :oracle  => {
               :default => {
                 :service => Chef::Provider::Service::Redhat,
