--- chef-11.12.2/lib/chef/platform/provider_mapping.rb	2014-05-03 16:17:20.832320849 +0300
+++ chef-11.12.2/lib/chef/platform/provider_mapping.rb	2014-05-03 16:16:41.990604626 +0300
@@ -185,6 +185,16 @@
               :group => Chef::Provider::Group::Usermod
             }
           },
+          :pld => {
+            :default => {
+              :service => Chef::Provider::Service::Redhat,
+              :cron => Chef::Provider::Cron,
+              :package => Chef::Provider::Package::Poldek,
+              :mdadm => Chef::Provider::Mdadm,
+              :ifconfig => Chef::Provider::Ifconfig::Redhat, # TODO: PLD config path needed
+              :group => Chef::Provider::Group::Usermod,
+            }
+          },
           :suse     => {
             :default => {
               :service => Chef::Provider::Service::Redhat,
